angularjs/angular.d.ts

```diff
@@ -708,7 +708,6 @@ declare module angular {
     // see http://docs.angularjs.org/api/ng.$window
     ///////////////////////////////////////////////////////////////////////////
     interface IWindowService extends Window {
-        [key: string]: any;
```

Comment:
People want this for *explicit* unsafe usage of `$window`. Shouldn't be removed :rose: